zulipterminal/ui_tools/boxes.py

```diff
@@ -401,7 +401,11 @@ def main_view(self) -> List[Any]:
                 ], dividechars=1)
         else:
             content_header = None
-
+        # If the message contains '/me' emote then replace it with
+        # user full name and show it in bold.
+        if self.message['is_me_message'] is True:
+            self.message['content'] = self.message['content'].replace(
+                '/me', '<strong>' + self.model.user_full_name + '</strong>')
```

Comment:
I'd specify `count=1` for the replace, just to be safe

Reply:
count=1 will simply do since we are already checking the value of `is_me_message`. We will follow webapp. count check will be necesarry in a message like this "/me is valid and \`/me\` is test."